pinotdb/db.py

```diff
@@ -147,6 +147,7 @@ def __init__(self, *args, **kwargs):
         if self.session:
             self.verify_session()
             self.is_session_external = True
+        self.timeout = kwargs.get('timeout', 'None')
```

Comment:
Same here.

if we are going to directly read `self._kwargs.get('timeout')`, we probably don't need to keep timeout in the self. If we want to keep the `self.timeout`, let's pass this when we initializing the `httpx.Client`

Reply:
On further inspection I found out that this is redundant code. Removing it.